ARCH/vpr_arch/k4_frac_N8_tileable_reset_softadder_register_scan_chain_nonLR_caravel_io_skywater130nm.xml

```diff
@@ -73,7 +73,7 @@
          These clocks can be handled in back-end
      -->
     <!-- Top-side has 1 I/O per tile -->
-    <tile name="io_top" capacity="1" area="0">
+    <tile name="io_top" capacity="32" area="0">
```

Comment:
This architecture file is used as a test chip architecture. As the architecture is frozen for the tape-outs. It is dangerous to modify the file. The I/O capacity is indeed designed for Caravel SoC. It is better to create a new architecture file for these changes.

Reply:
I have reverted changes to the base architecture